src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,48 +1,71 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+
+type Props = {
+  setModalTodo: React.Dispatch<React.SetStateAction<Todo | undefined>>,
```

Comment:
This type is so massive. It will be better to make a separate handler for this logic and pass a type of this handler here.

`setModalTodo: type of your handler`